github3/git.py

```diff
@@ -164,21 +168,43 @@ def _update_attributes(self, tag):
         super(Tag, self)._update_attributes(tag)
 
         #: String of the tag
-        self.tag = self._get_attribute(tag, 'tag')
+        self.tag = tag['tag']
 
         #: Commit message for the tag
-        self.message = self._get_attribute(tag, 'message')
+        self.message = tag['message']
 
         #: dict containing the name and email of the person
-        self.tagger = self._get_attribute(tag, 'tagger')
+        self.tagger = tag['tagger']
 
         #: :class:`GitObject <GitObject>` for the tag
-        self.object = self._class_attribute(tag, 'object', GitObject, self)
+        self.object = GitObject(tag['object'], self)
 
     def _repr(self):
         return '<Tag [{0}]>'.format(self.tag)
 
 
+class CommitTree(GitData):
+
+    """The :class:`CommitTree <CommitTree>` object.
+
+    Represents the tree data found in a commit object
+
+    """
+
+    def _update_attributes(self, tree):
+        super(CommitTree, self)._update_attributes(tree)
```

Comment:
This looks to be unnecessary. Were there extra attributes we wanted to include here?

Reply:
Hrm, this is probably just some copy pasta that got in from another place. I don't think it's necessary, or maybe I was in the middle of doing something and forgot to add them? I'll take another look at it.